.github/scripts/Update_Database.py

```diff
@@ -0,0 +1,111 @@
+from github import Github
+import json
+import sys
+import re
+
+# Regex Patterns
+category = r"- \[x\] (.+)"
+name = r"Title: (.+)"
+path = r"Folder: (.+)"
+requirments_path = r"Requirments: (.+)"
+entry = r"Script: (.+)"
+arguments = r"Arguments: (.+)"
+contributor = r"Contributor: (.+)"
+description = r"Description: (.+)"
+
+
+def add_script(category, name, path, entry, arguments, requirments_path, contributor, description, pa_token):
+    """ Add a Contributor script to database """
+    new_data = {category: {name: [path, entry, arguments, requirments_path, contributor, description]}}
+    data_store = read_data()
+    
+    try:
```

Comment:
Does this function also cover the case when the PR is a script and a imgbot PR or dependency PR? It should not make any commits to the repo when there is nothing to be updated in the database

Reply:
Umm, no.
But I think we can solve this issue on the workflow.yml file where we can validate the author of the repo.

Reply:
I think you call handle this by:
- Checking if the regex extracted fields are `""` empty. use Try-except for this.
- If that's the case, then you can terminate the script. 
- I am asking this because the `gh-pages` branch will have contributors that will not contribute to the scripts branch and their PR will not have any project meta data